sys/include/net/gnrc/ipv6/nc.h

```diff
@@ -88,7 +88,6 @@ extern "C" {
 #define GNRC_IPV6_NC_IS_ROUTER          (0x08)  /**< The neighbor is a router */
 
 #define GNRC_IPV6_NC_TYPE_MASK          (0x30)  /**< Mask for neighbor cache state */
-#define GNRC_IPV6_NC_TYPE_POS           (4)     /**< Shift of neighbor cache state */
```

Comment:
Ah okay… probably legacy code from the beginning of the API. Can be removed indeed.